buildstockbatch/gcp/main.tf

```diff
@@ -56,7 +61,17 @@ resource "google_storage_bucket" "bucket" {
 
 # Artifact registry repository
 resource "google_artifact_registry_repository" "AR_repo" {
```

Comment:
I tested this by creating a new `buildstockbatch-docker-test` repo [here](https://console.cloud.google.com/artifacts/docker/buildstockbatch-dev/us-central1/buildstockbatch-docker-test?project=buildstockbatch-dev).